Makefile

```diff
@@ -64,6 +64,8 @@ else
 ./envhelp/py3.depends: $(REQS_PATH)
 endif
 	@rm -f ./envhelp/py3.depends
+	@echo "upgrading venv pip as required for some dependencies"
+	@./envhelp/venv/bin/pip3 install --upgrade pip
```

Comment:
Realise this has already landed, but what requires a newer version of pip?

Reply:
Paramiko and in particular its dependencies won't build with the default old 9.x pip ... it basically turns into a long wild dependency goosechase if trying. Including various fixes to get setuptools and its more recent rust plugin installed _before_ installing additional dependencies. Been there, done that on `docker-migrid` as you can perhaps decipher in the Dockerfile.X in that repo.